crates/sozo/ops/src/migration/migrate.rs

```diff
@@ -775,6 +775,20 @@ pub async fn update_manifests_and_abis(
         rpc_url: rpc_url.to_string(),
     });
 
+    // Serialize the WorldMetadata to a string
+    let metadata = serde_json::to_string(&local_manifest.world.inner.metadata.as_ref().unwrap())?;
+    // let ipfs_hash = calculate_ipfs_hash(&local_manifest).await?;
+
+    // Create an ArtifactMetadata object from the serialized metadata
+    let artifact = ArtifactMetadata { abi: None, source: Some(serde_json::from_str(&metadata)?) };
```

Comment:
The ABI is actually included in the IPFS artifacts, we should do this check once we have the full content of the IPFS data.